### DISCUSSION TITLE
[DMC] WebView2 not loading web page in Outlook VSTO Add-in #1179

### DISCUSSION THREAD

**Discussion by Reezaali** [MicrosoftEdge/WebView2Feedback#3868] [Q&A] [answered]
Please review the customer issue located here: https://github.com/MicrosoftDocs/edge-developer/issues/1179

cc: @kumarkaran

**Comment by champnic** [comment, marked as answer]
Hey @kumarkaran -

Are you setting a user data folder explicitly? VSTO's using WebView2 need to specify a user data folder outside of the default, which for a VSTO is in Program Files where we don't have write permissions. More info here:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/userdatafolder
